CHANGES.rst

```diff
@@ -2,6 +2,7 @@
 ------------------
 
 - Add converter for the new ``Schechter1D`` model. [#67]
+- Add CITATION file. [#71]
```

Comment:
Not sure if this needs a change log.

(p.s. Not sure why it didn't go with the review submission.)